### DISCUSSION TITLE
I don't see data

### DISCUSSION THREAD

**Discussion by isycsl23** [djansen1987/SAJeSolar#63] [Q&A] [answered]
Hello, first of all I am new to this.
I have a Saj AS1-3KS-5.1 battery and AOI3 connected with a network cable.
I have installed the SAJ eSolar and I get the sensors, but they don't give me any value, it's as if they don't work.
Thank you

Sorry for the English it is through translation

**Comment by djansen1987** [comment]
Hi, What do the logs say? you can also enable debug via de configuration.yaml https://github.com/djansen1987/SAJeSolar#debugging

**Reply by isycsl23** [reply]
Hello
where do I see the records?
I put debugging in the .yaml and I don't see anything different
the sensors still do not give information
thank you

**Reply by djansen1987** [reply, marked as answer]
Settings->system->logs

**Comment by isycsl23** [comment]
hello
Este error se originó a partir de una integración personalizada.

Logger: custom_components.saj_esolar.sensor
Source: custom_components/saj_esolar/sensor.py:854
Integration: saj_esolar (documentation, issues)
First occurred: 18:06:08 (1804 occurrences)
Last logged: 18:26:09

return data NONE

Este error se originó a partir de una integración personalizada.

Logger: custom_components.saj_esolar.sensor
Source: custom_components/saj_esolar/sensor.py:853
Integration: saj_esolar (documentation, issues)
First occurred: 18:11:09 (4 occurrences)
Last logged: 18:26:09

Cannot poll eSolar using url: %s

Este error se originó a partir de una integración personalizada.

Logger: custom_components.saj_esolar.sensor
Source: custom_components/saj_esolar/sensor.py:514
Integration: saj_esolar (documentation, issues)
First occurred: 18:06:08 (1 occurrences)
Last logged: 18:06:08

Cannot poll eSolar using url: %s

**Reply by djansen1987** [reply]
Do you maybe have an installer account with multiple plants. If so, create an account that only has access to that plant

**Comment by isycsl23** [comment]
Registro completo:

[home-assistant_2023-09-02T16-28-21.087Z.log](https://github.com/djansen1987/SAJeSolar/files/12504323/home-assistant_2023-09-02T16-28-21.087Z.log)

**Comment by isycsl23** [comment]
Thanks for the help.
  but I can't create a new plant for myself.
The plant has access Saj--Seller--I
I am undergoing testing for a battery change under warranty.
For now I have to leave it that way.
Thank you very much for everything and sorry
